app/screens/entry/entry.js

```diff
@@ -202,11 +201,8 @@ export default class Entry extends PureComponent {
             const {data, text, badge, completed} = notificationData;
 
             // if the notification has a completed property it means that we are replying to a notification
-            // and in case it doesn't it means we just opened the notification
             if (completed) {
                 onPushNotificationReply(data, text, badge, completed);
-            } else {
-                await store.dispatch(loadFromPushNotification(notification));
```

Comment:
Why is this being removed?

Reply:
Not needed here, it was being called twice